tensortrade/oms/exchanges/exchange.py

```diff
@@ -40,7 +40,7 @@ class ExchangeOptions:
     """
 
     def __init__(self,
-                 commission: float = 0.003,
+                 commission: float = 0.001,
```

Comment:
Stop changing our defaults please.

Reply:
Version management. Ignore this

Reply:
That's not how it works. We can't ignore it. You need to back it out, along with all of your other bespoke stuff

Reply:
Same as this one: https://github.com/tensortrade-org/tensortrade/pull/430#discussion_r818912454